client/reader/sidebar/reader-sidebar-lists/list-item.jsx

```diff
@@ -45,23 +45,25 @@ export class ReaderSidebarListsListItem extends Component {
 		const { list, translate } = this.props;
 		const listRelativeUrl = `/read/list/${ list.owner }/${ list.slug }`;
 		const listManagementUrls = [
-			listRelativeUrl + '/tags',
+			listRelativeUrl + '/items',
 			listRelativeUrl + '/edit',
-			listRelativeUrl + '/sites',
+			listRelativeUrl + '/export',
+			listRelativeUrl + '/delete',
 		];
 
 		const lastPathSegment = last( this.props.path.split( '/' ) );
 		const isCurrentList =
 			lastPathSegment &&
+			// Prevents partial slug matches (e.g. bluefuton/test and bluefuton/test2)
 			lastPathSegment.toLowerCase() === list.slug.toLowerCase() &&
 			ReaderSidebarHelper.pathStartsWithOneOf( [ listRelativeUrl ], this.props.path );
-		const isActionButtonSelected = ReaderSidebarHelper.pathStartsWithOneOf(
```

Comment:
There's no longer an action button in the sidebar, so I've renamed the variable accordingly.